.github/workflows/refresh_db.yml

```diff
@@ -2,8 +2,8 @@ name: DB Refresh (Objects API)
 
 on:
   schedule:
-#    - cron: '*/20 * * * *'  # every 20 minutes
     - cron: '0 5 * * *'  # every day 5am GMT (12-1am EST/EDT)
+#    - cron: '*/20 * * * *'  # every 20 minutes
```

Comment:
### DB refresh cronjob _was_ working
By working, I mean that it wasn't simply exiting out with "refresh done; nothing to do" for no reason.

When we spoke earlier, I looked at several days of logs, and for those days, there actually were _no new csets created_; I confirmed.

I checked a 7 day period, and the results were: 
- stephanie's cset: should have found it, but didn't. problematic
- a 4 day period where there were *no* new csets, so it was correct when it said it had nothing to do
- before that, i checked the dates of csets created, and how many csets were gotten back from the refresh that day, and it got back the correct amount each day